lib/std/simd.zig

```diff
@@ -314,6 +325,7 @@ pub fn prefixScanWithFunc(
 /// Returns a vector whose elements are the result of performing the specified operation on the corresponding
 /// element of the input vector and every hop'th element that came before it (or after, if hop is negative).
 /// Supports the same operations as the @reduce() builtin. Takes O(logN) to compute.
+/// Note that the algorithm used by this function might affect floating point rounding errors.
```

Comment:
How does it affect floating point rounding errors? increase, decrease?
You could explain that its the programmer's job to ensure errors are in expected range + add mention that there is no function yet to capture those per platform.

Reply:
I meant it more as a warning for those who need deterministic behavior. Kinda like `@setFloatMode`.
In theory, this function should actually reduce rounding errors compared to a linear, sequential scan, as the values are more likely to be similar.